# Phylogeography and Evolutionary History of the Wood Frog (*Rana sylvatica*)

## Introduction and Goals

The wood frog (*Rana sylvatica*) is largely distributed all over North America, with northern limits in Alaska, and southern limits in Georgia and Alabama, and is the only frog species to have populations beyond the arctic circle (Martof & Humphries 1959). Wood frogs have an interesting post-glacial recolonization history: previous work using two mitochondrial genes determined that wood frogs comprise two distinct lineages: an eastern and western clade, and a maritime eastern subclade (Lee-Yaw et al. 2008). These two clades of wood frogs expanded their range northward and occupied colder environments. 

The goal of my project is reconstruct the evolutionary history of wood frogs through analyzing ddRADSeq data and using a phylogeographic approach. I will be collecting DNA from museum specimens and from new samples during collecting trips. I aim to explore how the two lineages of wood frogs diverged, how genetically isolated they are now, identify possible contact zones, and investigate whether the two lineages adapted to colder environment in similar (parallel) ways.
My project builds open previous findings by Lee-Yaw et al, 2008 and aims to fill several gaps in our understanding of the full recolonization history. Mainly, many populations and sites were not represented in the 2008 study, and we still lack knowledge of possible contact zones between wood frog populations. For example, a particular area of interest is the northern boundary between the eastern and western clades, which was not resolved in the study. 
Because I do not yet have ddRADseq data for wood frogs, I will be basing my project on a  published study that generated ddRADseq data for the Eurasian spadefoot toads (*Pelobates spp.*) (Dufresnes et al. 2019). This data is publicly available on NCBI as raw individual RAD sequences under BioProject PRJNA542138 with accessions SAMN11612144–SAMN11612336.

I will replicate the analyses the authors performed, starting with processing the raw sequence data and calling SNPs to generate sequence alignments and SNP matrices for downstream analyses. Similarly to the authors, I will perform Bayesian phylogenetics reconstructions of the SNP dataset, and perform PCA to visualize the genetic structure among the populations of spadefoots. I will use the Bayesian clustering algorithm of the program STRUCTRE to infer population differentiation. Finally, I will analyze the contact zones between three pairs of taxa by exploring the genetic admixture between them.

This project will enable me to develop an analysis pipeline to use for the future ddRADseq data that I will be generating from wood frog samples.

## Methods

The tools I used were... See analysis files at (links to analysis files).

## Results

The tree in Figure 1...

## Discussion

These results indicate...

The biggest difficulty in implementing these analyses was...

If I did these analyses again, I would...

## References

Dufresnes C, Strachinis I, Suriadna N, Mykytynets G, Cogălniceanu D, et al. 2019. Phylogeography of a cryptic speciation continuum in Eurasian spadefoot toads ( Pelobates ). Mol Ecol. 28(13):3257–70. https://onlinelibrary.wiley.com/doi/abs/10.1111/mec.15133

Lee-Yaw JA, Irwin JT, Green DM. 2008. Postglacial range expansion from northern refugia by the wood frog, Rana sylvatica. Mol. Ecol. 17(3):867–84. https://pubmed.ncbi.nlm.nih.gov/18179428/

Martof BS, Humphries RL. 1959. Geographic Variation in the Wood Frog Rana sylvatica. American Midland Naturalist. 61(2):350. https://www.jstor.org/stable/pdf/2422506.pdf


#this is a test 2

